components/automate-gateway/handler/notifications.go

```diff
@@ -148,7 +149,7 @@ func (s *notificationsServer) ValidateWebhook(ctx context.Context,
 
 	resp, err := s.notificationsClient.ValidateWebhook(ctx, &transformUVR)
 	if err != nil {
-		return nil, err
+		return nil, errors.New("something went wrong")
```

Comment:
I don't think the users will get benefit from this error message. Error message is not appropriate

Reply:
Updated msg